src/views/tickets/view-link.php

```diff
@@ -40,12 +40,20 @@
 
 $counters = [];
 
-if ( 0 !== $rsvp_count ) {
-	$counters[] = sprintf( _n( '%d RSVP', '%d RSVPs', $rsvp_count, 'event-tickets' ), $rsvp_count );
+if ( 1 === $rsvp_count ) {
+	$counters[] = sprintf( _x( '%d %s', 'RSVP count singular', 'event-tickets' ), $rsvp_count, tribe_get_rsvp_label_singular( basename( __FILE__ ) ) );
 }
 
-if ( 0 !== $ticket_count ) {
-	$counters[] = sprintf( _n( '%d Ticket', '%d Tickets', $ticket_count, 'event-tickets' ), $ticket_count );
+if ( 1 < $rsvp_count ) {
+	$counters[] = sprintf( _x( '%d %s', 'RSVP count plural', 'event-tickets' ), $rsvp_count, tribe_get_rsvp_label_plural( basename( __FILE__ ) ) );
+}
+
+if ( 1 === $ticket_count ) {
+	$counters[] = sprintf( _x( '%d %s', 'Ticket count singular', 'event-tickets' ), $ticket_count, tribe_get_ticket_label_singular( basename( __FILE__ ) ) );
+}
+
+if ( 1 < $ticket_count ) {
+	$counters[] = sprintf( _x( '%d %s', 'Ticket count plural', 'event-tickets' ), $ticket_count, tribe_get_ticket_label_plural( basename( __FILE__ ) ) );
 }
 
 $link = $view->get_tickets_page_url( $event_id, $is_event_page );
```

Comment:
On the line 61 below, it should be escaping the whole `sprintf()`, and then you can change `esc_html__()` to a plain `__()`

Reply:
@cliffordp this also needs a resolution

Reply:
@cliffordp can you tackle this escaping fix as part of this PR?

Reply:
```php
$message  = esc_html( sprintf( __( 'You have %s for this %s.', 'event-tickets' ), implode( __( ' and ', 'event-tickets' ), $counters ), $events_label_singular ) );

$message .= ' <a href="' . esc_url( $link ) . '">' . esc_html( sprintf( __( 'View your %s', 'event-tickets' ), $this->get_description_rsvp_ticket( $event_id, $user_id ) ) ) . '</a>';
```